sdk/eventhubs/azure-messaging-eventhubs/src/main/java/com/azure/messaging/eventhubs/EventHubProducerAsyncClient.java

```diff
@@ -93,6 +94,7 @@
 
     private static final SendOptions DEFAULT_SEND_OPTIONS = new SendOptions();
     private static final CreateBatchOptions DEFAULT_BATCH_OPTIONS = new CreateBatchOptions();
+    private static final String AZ_NAMESPACE_VALUE = "Microsoft.EventHub";
```

Comment:
Is there a constants class in the implementation package we can put this in, instead of triplicating the constant?

Reply:
added it to the client constants file.